BGL/test/BGL/CMakeLists.txt

```diff
@@ -71,8 +71,6 @@ create_single_source_cgal_program( "graph_concept_Polyhedron_3.cpp" )
 
 create_single_source_cgal_program( "graph_concept_Dual.cpp" )
 
-create_single_source_cgal_program( "graph_concept_Triangulation_2.cpp" )
```

Comment:
That commit has removed `graph_concept_Triangulation_2.cpp` from the `CMakeLists.txt`, and that was the first `test_Triangulation_2.cpp` that was removed!!